nixos/modules/services/networking/gnunet.nix

```diff
@@ -130,16 +136,16 @@ in
 
     # The user tools that talk to `gnunetd' should come from the same source,
     # so install them globally.
-    environment.systemPackages = [ pkgs.gnunet ];
+    environment.systemPackages = [ config.services.gnunet.package ];
 
     systemd.services.gnunet = {
       description = "GNUnet";
       after = [ "network.target" ];
       wantedBy = [ "multi-user.target" ];
-      path = [ pkgs.gnunet pkgs.miniupnpc ];
+      path = [ config.services.gnunet.package pkgs.miniupnpc ];
```

Comment:
Is the gnunet package really needed in here?

Reply:
@viric put it there to get upnp working. https://github.com/NixOS/nixpkgs/commit/4f71bce69150bdea42a9374663e1eaaa5a40b7fe